docs/2.8.0/docs/kubernetes/04-deploy-daml-enterprise/01-helm-deployment/04-daml-http-json-deployment.rst

```diff
@@ -0,0 +1,94 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Install ``daml-http-json`` Chart
+################################
+
+Steps
+*****
+
+1. Parameterize the Helm chart
+==============================
+
+.. note::
+   The Terraform scripts were used to parameterize the Helm values. If standalone Helm deployment is done without Terraform the value file has to be customized manually. The template value file can be found in ``http-json.yaml``.
```

Comment:
I copied the http-json.yaml file and put it in the docs repo. Need to figure out how to link to it.